client/components/Title.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
I think technically this statement should be removed until/unless you actually implement react in the file.

Reply:
https://medium.com/@Zwenza/why-do-you-need-to-import-react-in-functional-components-7385e4329ffb
https://stackoverflow.com/questions/51388723/why-should-we-use-import-react-from-react
Babel will transform this in the future so I will need to import.